tests/Http/ImagesTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Tests\Http;
+
+use Tests\TestCase;
+use Rogue\Models\Post;
+
+class ImagesTest extends TestCase
+{
+    /**
+     * Test for endpoint throttling.
+     *
+     * @return void
+     */
+    public function testImagesThrottle()
+    {
+        // Make a post to view
+        $post = factory(Post::class)->create();
+
+        // View the post 15 times (using 3 different versions)
+        for ($i = 0; $i < 5; $i++) {
```

Comment:
Just making sure -  this makes the requests fast enough to model an aggressive user loading too many images?

Reply:
@sbsmith86 I believe so since the test takes way less than a minute to run. Here is the test in Northstar that it's modeled after: https://github.com/DoSomething/northstar/blob/5a98445784fa94c5fc0a80a92f66f9621293b200/tests/Http/Web/WebAuthenticationTest.php#L231-L245

Reply:
Yeah, this for-loop should run incredibly quickly.